prisma/schema.prisma

```diff
@@ -13,21 +13,24 @@ generator client {
 model Transfer {
   id           String         @id @default(auto()) @map("_id") @db.ObjectId
   depositNonce Int            @unique
-  resource     Resource       @relation(fields: [resourceID], references: [resourceId])
-  resourceID   String
-  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [domainId])
+  resource     Resource?       @relation(fields: [resourceID], references: [id])
+  resourceID   String?
+  fromDomain   Domain         @relation(name: "fromDomainRelation", fields: [fromDomainId], references: [id])
   fromDomainId String
-  toDomain     Domain         @relation(name: "toDomainRelation", fields: [toDomainId], references: [domainId])
-  toDomainId   String
-  type         String
-  sender       String
-  amount       String
-  destination  String
+  toDomain     Domain?        @relation(name: "toDomainRelation", fields: [toDomainId], references: [id])
+  toDomainId   String?
+  sender       String?
+  destination  String?
+  amount       String?
   status       TransferStatus @default(pending)
   fee          Fee?
   deposit      Deposit?
   execution    Execution?
   timestamp    Int?
+  ofacComply   Boolean?
```

Comment:
@MakMuftic I wonder if here we should use something like `isSanctioned`... now I wonder